substrate/frame/assets/src/types.rs

```diff
@@ -232,6 +232,32 @@ impl<AssetId, AccountId, Balance> FrozenBalance<AssetId, AccountId, Balance> for
 	fn died(_: AssetId, _: &AccountId) {}
 }
 
+/// Trait for specifying a balance that is distinct from the free balance.
+/// This balance is then summed up with the balance of the account, and the
+/// `minimum_balance` (and frozen balance, if any) of the asset to calculate
+/// the reducible balance.
```

Comment:
account.balance + minimum_balance + frozen balance = reducible balance?

Reply:
Need to update the definition of the trait to reflect how this works.